config/config-example.js

```diff
@@ -210,6 +210,11 @@ exports.appealurl = '';
 exports.replsocketprefix = './logs/repl/';
 exports.replsocketmode = 0o600;
 
+// ripgrepmodlog - whether to use ripgrep to handle modlog queries
+// If no value is set, the first modlog query after each restart will check if
+// ripgrep binaries are available and, if so, they will be used
+exports.ripgrepmodlog = undefined;
```

Comment:
`false` = don't use ripgrep, don't check if it's there
`true` = use ripgrep, don't check if it's there
`undefined` = check if ripgrep is available and only use it if it's there (consequently sets the flag accordingly)
I don't think it needs to check every time modlog is used, so a third value is required.